doc/devices/gbs.rst

```diff
@@ -0,0 +1,160 @@
+The GBS device
+==============
+
+The GBS device gives access to a near-term model of photonic quantum computing called `Gaussian
+boson sampling <https://strawberryfields.ai/photonics/concepts/gbs.html>`__. This model can
+encode graphs and other quantities that can be represented by a symmetric matrix :math:`A`, with
```

Comment:
Maybe good here to also mention the mean photon number. Otherwise it appears as a bit of a surprise in line 44